ocean_lib/web3_internal/contract_utils.py

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import importlib
+import json
+import logging
+import os
+from collections import namedtuple
+from decimal import Decimal
+from pathlib import Path
+
+import artifacts
+from enforce_typing import enforce_types
+from eth_account.messages import encode_defunct
+from eth_keys import keys
+from eth_utils import big_endian_to_int, decode_hex
+from jsonsempai import magic  # noqa: F401
+from web3 import Web3
+from web3.contract import ConciseContract
+
+from ocean_lib.web3_internal.constants import DEFAULT_NETWORK_NAME, NETWORK_NAME_MAP
+from ocean_lib.web3_internal.web3_overrides.signature import SignatureFix
+from ocean_lib.web3_internal.web3_provider import Web3Provider
+
+Signature = namedtuple("Signature", ("v", "r", "s"))
+
+logger = logging.getLogger(__name__)
+
+
+def get_contract_definition(contract_name):
+    try:
+        return importlib.import_module("artifacts." + contract_name).__dict__
+    except ModuleNotFoundError:
+        raise TypeError("Contract name does not exist in artifacts.")
+
+
+def load_contract(web3, contract_name, address):
```

Comment:
I think it should stay this way, since the others are "regular" utils, and "load" is more descriptive and high-level. I would keep it as such. The "get_contract_concise" is the outlier, actually, but that is set to be deprecated.